### DISCUSSION TITLE
Should we add a integrated profiler?

### DISCUSSION THREAD

**Discussion by roigcarlo** [KratosMultiphysics/Kratos#7955] [Ideas]
We could integrate support for the @profile decorator in the Kratos init so people can easily use some python profilers without having to launch the scripts form custom launchers or modify the code.

What do you think @KratosMultiphysics/technical-committee ?

**Comment by philbucher** [comment]
if it is not too much effort I would be in favor

**Comment by pooyan-dadvand** [comment]
I am in favor!! :+1:
